sdk/communication/Azure.Communication.Chat/src/ChatThreadClient.cs

```diff
@@ -120,7 +120,7 @@ public virtual async Task<Response<string>> SendMessageAsync(string content, Cha
         /// <param name="senderDisplayName"> The display name of the message sender. This property is used to populate sender name for push notifications. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
-        public virtual Response<string> SendMessage(string content, ChatMessageType? type = null, string? senderDisplayName = null, CancellationToken cancellationToken = default)
+        public virtual Response<string> SendMessage(string content, ChatMessageType? type = null, string senderDisplayName = null, CancellationToken cancellationToken = default)
```

Comment:
I think we can just use ChatMessageType type = default here

Reply:
changed

Reply:
Also the nullabilty is not needed it I believe :
ChatMessageType type = default instead of 
ChatMessageType? type = default

Reply:
you are right, I've updated it :)